tasks/trace_agent.py

```diff
@@ -55,6 +55,8 @@ def build(ctx, rebuild=False, race=False, precompile_only=False, use_embedded_li
         "REPO_PATH": REPO_PATH,
     }
 
+    print(env)
```

Comment:
can we remove these print statements?

Reply:
I am using one to re-launch similar actions from the console